src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -570,4 +477,29 @@ private void initializeJmxSeedsForAllClusters() {
       LOG.info("Initialized JMX seed list for all clusters.");
     }
   }
+
+  private void tryInitializeStorage(ReaperApplicationConfiguration config, Environment environment)
+    throws ReaperException, InterruptedException {
+    if (context.storage == null) {
+      LOG.info("initializing storage of type: {}", config.getStorageType());
+      int storageFailures = 0;
+      while (true) {
+        try {
+          context.storage = InitializeStorage.initializeStorage(
+            config, environment, context.reaperInstanceId).initializeStorageBackend();
+          break;
+        } catch (RuntimeException e) {
+          LOG.error("Storage is not ready yet, trying again to connect shortly...", e);
+          storageFailures++;
+          if (storageFailures > 60) {
+            LOG.error("Too many failures when trying to connect storage. Exiting :'(");
+            System.exit(1);
```

Comment:
would exception handling at the command level be more appropriate place to do the `exit(..)` ? 
(that way you can easily list, and read, the different exit codes that exist)

Reply:
done 👍